### PR TITLE
QA fixes for phase 1

### PR DESCRIPTION
**Pull request by DhaaraniCIT**
## Summary by CodeRabbit

- **New Features**
	- Updated onboarding process for Sage 300 integration with new field labels and placeholders to reflect HH2 credentials.
	- Implemented additional attribute import functions in the Sage 300 onboarding connector.

- **Documentation**
	- Clarified the requirement for HH2 Identifier in the onboarding content text.

- **Style**
	- Adjusted padding and text color in various components for improved visual consistency.
	- Temporarily removed display of certain "Other Preferences" settings.

- **Bug Fixes**
	- Fixed conditional rendering logic for an icon in the configuration select field based on the `isDisabled` state.
	- Corrected the text color in the onboarding header to reflect the current status of integration setup.

### CODE REVIEW

**Comment by ashwin1111** [comment]
@coderabbitai review

File: src/app/integrations/sage300/sage300-onboarding/sage300-onboarding-connector/sage300-onboarding-connector.component.ts [hunk 1]
```diff
@@ -55,6 +57,8 @@ export class Sage300OnboardingConnectorComponent implements OnInit {
       this.isLoading = false;
       this.toastService.displayToastMessage(ToastSeverity.SUCCESS, 'Connection Successful.');
       this.workspaceService.setOnboardingState(Sage300OnboardingState.EXPORT_SETTINGS);
+      this.workspaceService.importFyleAttributes(false).subscribe();
```
Comment (ashwin1111) [inline src/app/integrations/sage300/sage300-onboarding/sage300-onboarding-connector/sage300-onboarding-connector.component.ts:60]: this is not required

File: src/app/integrations/sage300/sage300-onboarding/sage300-onboarding-connector/sage300-onboarding-connector.component.ts [hunk 2]
```diff
@@ -55,6 +57,8 @@ export class Sage300OnboardingConnectorComponent implements OnInit {
...
+      this.mappingService.importSage300Attributes(false).subscribe();
```
Comment (ashwin1111) [inline src/app/integrations/sage300/sage300-onboarding/sage300-onboarding-connector/sage300-onboarding-connector.component.ts:61]: send true to be safe

File: src/app/integrations/sage300/sage300-shared/sage300-advanced-settings/sage300-advanced-settings.component.html [hunk 3]
```diff
@@ -87,7 +87,7 @@ <h4 class="tw-mb-12-px">Preview of the Description Field</h4>
                     </app-configuration-skip-export>
                 </div>
             </div>
-            <div class="tw-mb-16-px">
+            <!-- <div class="tw-mb-16-px">
```
Comment (ashwin1111) [inline src/app/integrations/sage300/sage300-shared/sage300-advanced-settings/sage300-advanced-settings.component.html:90]: why is this commented? is this not being released for v1? can you point me to the thread

Comment (DhaaraniCIT) [reply src/app/integrations/sage300/sage300-shared/sage300-advanced-settings/sage300-advanced-settings.component.html:90]: https://fylein.slack.com/archives/C050P7PH492/p1702538418152329?thread_ts=1702538023.176009&cid=C050P7PH492

Comment (DhaaraniCIT) [reply src/app/integrations/sage300/sage300-shared/sage300-advanced-settings/sage300-advanced-settings.component.html:90]: Screenshot 2023-12-18 at 4 56 58 PM

Comment (ashwin1111) [reply src/app/integrations/sage300/sage300-shared/sage300-advanced-settings/sage300-advanced-settings.component.html:90]: got it

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**